deploy/docker/cp-release-notes/config/application.properties

```diff
@@ -0,0 +1,40 @@
+spring.main.banner-mode=off
+
+pipeline.api.url=https://${CP_API_SRV_INTERNAL_HOST:cp-api-srv.default.svc.cluster.local}:${CP_API_SRV_INTERNAL_PORT:31080}/pipeline/restapi/
+pipeline.api.token=${CP_API_JWT_ADMIN}
+pipeline.client.connect.timeout=30
+pipeline.client.read.timeout=30
+pipeline.api.version.file.path=${CP_RELEASE_NOTES_VERSION_FILE_PATH:/opt/release-notes/data/version}
+
+github.baseurl=${CP_RELEASE_NOTES_GITHUB_BASE_URL:https://api.github.com}
+github.default.branch.name=develop
+github.owner.name=epam
+github.read.timeout=30
+github.connect.timeout=30
+github.project.name=cloud-pipeline
+github.issue.regex=(?i)\\(?issue #.+
+github.issue.number.regex=.+#(\\d+).*
+github.token=${CP_RELEASE_NOTES_GITHUB_TOKEN}
+
+jira.base.url=${CP_RELEASE_NOTES_JIRA_URL}
+jira.auth.token=Basic ${CP_RELEASE_NOTES_JIRA_BASE_64_TOKEN}
+jira.version.custom.field.id=${CP_RELEASE_NOTES_JIRA_VERSION_FIELD_ID:}
+jira.github.custom.field.id=${CP_RELEASE_NOTES_JIRA_GITHUB_FIELD_ID:}
+
+release.notes.agent.admin.emails=${CP_RELEASE_NOTES_ADMIN_EMAIL:}
```

Comment:
now I see that this two case (admin/subscribers) actually very similar cases and it differ from each other only in matter of version change type (minor/major)

so actually I thing it would be much better to refactor this names to:
```
release.notes.agent.templates.dir=

# Major version changed case's settings
release.notes.agent.major.version.changed.subscriber.emails=
release.notes.agent.major.version.changed.email.template=
release.notes.agent.major.version.changed.subject=

# Minor version changed case's settings
release.notes.agent.minor.version.changed.subscriber.emails=
release.notes.agent.minor.version.changed.email.template=
release.notes.agent.minor.version.changed.subject=
```

Reply:
the same changes i believe better to do with env vars